### PR TITLE
Databasae functions to support functionalety we discussed in the meet…

### PR DESCRIPTION
**Pull request by MCunningham211**
Temporary database user to support OAuth sign-in with the front-end. User OAuth info is stored so that it can be polled by the front end later

### CODE REVIEW

File: src/database.ts [hunk 1]
```diff
@@ -189,4 +189,52 @@ export async function PollTokensFromDatabase(tenantAllias: string) {
...
+
+
+
+/**
+ * Used to store temporary user information
+ * @param hubspotID 
+ * @param accessToken 
+ * @param refreshToken 
+ */
+export function AddTempUser(hubspotID: string, accessToken = "", refreshToken = ""}) {
```
Comment (nicolepeverley) [inline src/database.ts]: Looks like you have an extra } bracket here

File: src/database.ts [hunk 2]
```diff
@@ -189,4 +189,52 @@ export async function PollTokensFromDatabase(tenantAllias: string) {
...
     });
     return data;
+}
+
+
+
+
+
+/**
+ * Used to store temporary user information
```
Comment (nicolepeverley) [inline src/database.ts:199]: Should we remove the AddTokensToDatabase function if this will now use the AddTempUser function instead?

**Comment by nicolepeverley** [comment]
[Create Temp Database Table for unconfigured users](https://trello.com/c/apFe2Csb/127-create-temp-database-table-for-unconfigured-users)

File: src/database.ts (hunk 2 above)
Comment (MCunningham211) [reply src/database.ts:199]: I don't think it can hurt to have it in at the moment

**Review by ChristopherRDunn — APPROVED** [review]

**Review by nicolepeverley — APPROVED** [review]
Looks good to me!

**Pull request merged**